### DISCUSSION TITLE
More instances running

### DISCUSSION THREAD

**Discussion by tiomanisland** [leokhoa/laragon#929] [General]
On Windows 11 if I click Laragon icon again another Laragon instance will be running so there will be more icons in tray.  Then webserver stops working. Wouldn't it make sense to check if Laragon is already running, click on icon should just open it instead of starting another instance?

**Comment by Pen-y-Fan** [comment]
This is a feature of Laragon, due to it being portable, see my blog [How to set up Laragon on a new Windows computer (part 3) - System tray icon](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-3/) for details on how to pin the Laragon icon, making it easier to spot when opened.

**Comment by tiomanisland** [comment]
I didn't ask about pinning the icon. I said you can click icon 3 times and then 3 instances of Laragon will be running. This is not ok, only one instance should be running.

**Comment by ppfoong** [comment]
Yes, I agree.

When Laragon starts, it can check for itself in existing Windows processes.

If found, it can prompt "Another Laragon is already running" and then exit.
